lib/manager/components/validation/MobilityDataValidationResult.js

```diff
@@ -0,0 +1,149 @@
+              )
+            })}
+          </tr>
+        ))}
+      </tbody>
+    </Table>
+  )
+}
+
+// eslint-disable-next-line complexity
+const renderNoticeDetail = (notice) => {
+  switch (notice.code) {
+    case 'too_many_rows':
+      notice.csvRowNumber = notice.rowNumber
+    case 'fare_transfer_rule_duration_limit_type_without_duration_limit':
+    case 'fare_transfer_rule_duration_limit_without_type':
+    case 'fare_transfer_rule_missing_transfer_count':
+    case 'fare_transfer_rule_with_forbidden_transfer_count':
+      notice.filename = 'fare_transfer_rules.txt'
+    case 'empty_file':
+    case 'emtpy_row':
+    case 'missing_timepoint_column':
+    case 'missing_required_file':
+    case 'missing_recommended_file':
+    case 'unknown_file':
+      return (
+        <ul style={{marginTop: '-10px'}}>
+          {notice.sampleNotices.map((notice) => (
+            <li>
+              {notice.filename}
+              {notice.csvRowNumber && `: row ${notice.csvRowNumber}`}
+            </li>
+          ))}
+        </ul>
+      )
+    default:
+      return (
+        <NoticeTable notices={notice.sampleNotices} />
+      )
+  }
+}
+
+const MobilityDataValidationResult = (props) => {
+  const { notice } = props
+  const rule = rules.find((rd) => rd.rule === notice.code)
+  if (!rule) return null
+
+  const errorClass = `gtfs-error-${mobilityDataValidationErrorMapping[notice.severity]}`
+  const [expanded, setExpanded] = useState(notice.totalNotices < 2)
+
+  const onRowSelect = () => setExpanded(!expanded)
+
+  return (
+    <ListGroupItem style={{ overflow: 'scroll', padding: 0 }}>
+      <div style={{ padding: '10px 15px' }}>
+        <h4
+          onClick={onRowSelect}
+          onKeyDown={onRowSelect}
+          style={{ cursor: 'pointer' }}
+        >
+          <span
+            className={`buffer-icon ${errorClass}`}
+            title={`${toSentenceCase(notice.severity)} priority`}
+          >
+            <Icon
+              type={validationErrorIconLookup[mobilityDataValidationErrorMapping[notice.severity]]}
+            />
+          </span>
+          {toSentenceCase(notice.code.replaceAll('_', ' ').toLowerCase())}
+          <span className={errorClass}>
+            {' '}
+            &mdash; {notice.totalNotices} case
+            {notice.totalNotices > 1 ? 's' : ''} found
+          </span>
+          <span className={`pull-right`}>
+            <Icon type={expanded ? 'caret-up' : 'caret-down'} />
+          </span>
+        </h4>
+        {expanded && (
+          <>
+            <p><Markdown>{rule.description}</Markdown></p>
+            <p>
+              <a
+                href={`https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#${notice.code}`}
+                target='_blank'
+                rel='noreferrer'
+              >
+                More details
+              </a>
+            </p>
+          </>
+        )}
+      </div>
+      {expanded && renderNoticeDetail(notice)}
```

Comment:
```suggestion
      {expanded && <div style={{overflow: 'scroll'}}>{renderNoticeDetail(notice)}</div>}
```
To apply the scroll only to this